resources/assets/uad_lists.json

```
@@ -17090,6 +17090,15 @@
     "labels": [],
     "removal": "Recommended"
   },
+  {
+    "id": "com.google.apps.pixelmigrate",
+    "list": "Google",
+    "description": "Migrate data to your new pixel device with a usb-c cable or whatever. Best removed. Apparently it doesn't\nwork very well anyway.\n\n(https://play.google.com/store/apps/details?id=com.google.android.apps.pixelmigrate)",
```

```suggestion
    "description": "Migrate data to your new Pixel device with a USB-C cable or whatever. Best removed. Apparently it doesn't work very well anyway.\nhttps://play.google.com/store/apps/details?id=com.google.android.apps.pixelmigrate",
```
What do you mean with "best removed"?

---

By "best removed", I meant "It is best to remove this".